src/fontra/client/core/experimental-features.js

```diff
@@ -0,0 +1,11 @@
+import { ObservableController } from "./observable-object.js";
+
+export const experimentalFeaturesController = new ObservableController({
```

Comment:
I would love to have this bundled with the clipboard settings as `applicationSettingsController`, even if that means we're changing the localStorage.

And either way, I'd like to drop the "Experimental" bit.

So, `clipboard-format.js` and `experimental-features.js` should together become `application-settings.js`, which should export a single `applicationSettingsController` object.

Reply:
Thanks for the great feedback. I am totally with you. I don't wanted to change too much, while we keep the 'gear'-sidebar. But now this is done.